### PR TITLE
[REVIEW] Updating docs

### LINKED ISSUE #725: [ENH] Redo and improve Python Docs
Our API docs could be improved, and we need to fill in all the non-API docs

### PR DESCRIPTION
**Pull request by BradReesWork**
Updating all the python docs to read better and use "from_cudf_edgelist"

closes #725

### CODE REVIEW

**Comment by BradReesWork** [comment]
rerun tests

File: python/cugraph/community/subgraph_extraction.py [hunk 1]
```diff
@@ -36,12 +36,12 @@ def subgraph(G, vertices):
...
-    >>>                   dtype=['int32', 'int32', 'float32'], header=None)
-    >>> sources = cudf.Series(M['0'])
-    >>> destinations = cudf.Series(M['1'])
+    >>> M = cudf.read_csv('datasets/karate.csv',
+                          delimiter = ' ',
+                          dtype=['int32', 'int32', 'float32'],
+                          header=None)
     >>> G = cugraph.Graph()
-    >>> G.add_edge_list(sources, destinations, None)
+    >>> G.from_cudf_edgelist(M, source='0', destination='1', edge_attr=None)
```
Comment (Iroy30) [inline python/cugraph/community/subgraph_extraction.py:44]: Don't need to explicitly mention edge_attr=None as it is default None. But this works too

File: python/cugraph/link_prediction/wjaccard.py [hunk 2]
```diff
@@ -69,7 +69,7 @@ def jaccard_w(input_graph, weights, vertex_pair=None):
     >>> weights = cudf.Series(numpy.ones(
     >>>     max(sources.max(),destinations.max())+1, dtype=numpy.float32))
```
Comment (Iroy30) [inline python/cugraph/link_prediction/wjaccard.py:70]: 67-69 not needed

File: python/cugraph/link_prediction/woverlap.py [hunk 3]
```diff
@@ -68,7 +68,7 @@ def overlap_w(input_graph, weights, vertex_pair=None):
     >>> weights = cudf.Series(numpy.ones(
```
Comment (Iroy30) [inline python/cugraph/link_prediction/woverlap.py:68]: 66-68 not needed

File: python/cugraph/traversal/sssp.py [hunk 4]
```diff
@@ -47,10 +47,8 @@ def sssp(G, source):
     --------
     >>> M = cudf.read_csv('datasets/karate.csv', delimiter=' ',
     >>>                   dtype=['int32', 'int32', 'float32'], header=None)
-    >>> sources = cudf.Series(M['0'])
-    >>> destinations = cudf.Series(M['1'])
     >>> G = cugraph.Graph()
-    >>> G.add_edge_list(sources, destinations, None)
+    >>> G.from_cudf_edgelist(gdf, source='0', destination='1')
```
Comment (Iroy30) [inline python/cugraph/traversal/sssp.py:51]: M instead of gif

File: python/cugraph/traversal/bfs.py [hunk 5]
```diff
@@ -43,10 +43,8 @@ def bfs(G, start):
     --------
     >>> M = cudf.read_csv('datasets/karate.csv', delimiter=' ',
     >>>                   dtype=['int32', 'int32', 'float32'], header=None)
-    >>> sources = cudf.Series(M['0'])
-    >>> destinations = cudf.Series(M['1'])
     >>> G = cugraph.Graph()
-    >>> G.add_edge_list(sources, destinations, None)
+    >>> G.from_cudf_edgelist(gdf, source='0', destination='1')
```
Comment (Iroy30) [inline python/cugraph/traversal/bfs.py:47]: M instead of gif

File: python/cugraph/community/subgraph_extraction.py (hunk 1 above)
Comment (BradReesWork) [reply python/cugraph/community/subgraph_extraction.py]: fixed

File: python/cugraph/link_prediction/wjaccard.py (hunk 2 above)
Comment (BradReesWork) [reply python/cugraph/link_prediction/wjaccard.py:70]: The weights are needed "df = cugraph.jaccard_w(G, weights)"

Comment (BradReesWork) [reply python/cugraph/link_prediction/wjaccard.py]: fixed

File: python/cugraph/link_prediction/woverlap.py (hunk 3 above)
Comment (BradReesWork) [reply python/cugraph/link_prediction/woverlap.py]: foxed

File: python/cugraph/traversal/sssp.py (hunk 4 above)
Comment (BradReesWork) [reply python/cugraph/traversal/sssp.py:51]: fixed

**Pull request merged**